### PR TITLE
VAN-3787 add demo apps that work with buildpacks

### PR DESCRIPTION
**Pull request by kanaksinghal**
I wanted a simple app that can be used to demonstrate value of Code Pipes.

01-nodejs is basic nodejs app - [readme.md](https://github.com/cldcvr/codepipes-tutorials/tree/kanak/VAN-3787/apps/01-nodejs)
02-nodejs-ts is nodejs app using typescript - [readme.md](https://github.com/cldcvr/codepipes-tutorials/tree/kanak/VAN-3787/apps/02-nodejs-ts)
03-nodejs-redis is nodejs typescript app using redis. - [readme.md](https://github.com/cldcvr/codepipes-tutorials/tree/kanak/VAN-3787/apps/03-nodejs-redis)

### CODE REVIEW

File: apps/02-nodejs-ts/readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+# using paketo
+# final image size ~270MB
+pack build 02-nodejs-ts:paketo --builder paketobuildpacks/builder:base
+
+# OR
+# using heroku
+# final image size ~910MB
+pack build 01-modejs:heroku --builder heroku/builder:22
```
Comment (dshelley66) [inline apps/02-nodejs-ts/readme.md:38]: should this be 01-nodejs? (same in other 2 readmes)

File: apps/02-nodejs-ts/readme.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Following environment variables can be configured for the application:
+
+```
+NODE_ENV=development
+```
+
+## Container image
+
+Container image can be build for this app automatically using buildpack builders from paketo or heroku. Or the Dockerfile can also be used to build directly using docker.
```
Comment (dshelley66) [inline apps/02-nodejs-ts/readme.md:28]: "..can be built...."

**Comment by dshelley66** [comment]
maybe someone from the frontend team can review? @NikhilVerma @Nomercy10 @shubhamseervi @kusumratawacld

**Comment by kanaksinghal** [comment]
even @krunalrasik can review design of the page.

File: apps/02-nodejs-ts/readme.md (hunk 1 above)
Comment (kanaksinghal) [reply apps/02-nodejs-ts/readme.md]: done 😅

File: apps/02-nodejs-ts/readme.md (hunk 2 above)
Comment (kanaksinghal) [reply apps/02-nodejs-ts/readme.md]: done 😅

**Review by pranaySinghDev — APPROVED** [review]

**Pull request merged**